astropy/table/tests/test_mixin.py

```diff
@@ -131,18 +131,74 @@ def test_io_quantity_write(tmpdir):
         assert 'cannot write table with mixin column(s)' in str(err.value)
 
 
+@pytest.mark.parametrize('table_types', (Table, QTable))
+def test_io_time_write_fits(tmpdir, table_types):
```

Comment:
What about testing the round-trip of a multi-dimensional time object (i.e. where `tm.shape`.  Is this supported by the code?